examples/using-emotion/src/pages/index.js

```diff
@@ -52,28 +43,29 @@ const title = css`
   }
 `
 
-// Using css with object
-const subtitle = css({
-  color: `#bd93f9`,
-})
-
 const IndexPage = () => (
   <Fragment>
     <Helmet>
       <title>Gatsby Emotion</title>
       <meta name="description" content="Gatsby example site using Emotion" />
       <meta name="referrer" content="origin" />
     </Helmet>
+    <Global styles={globalStyles} />
     <Wrapper>
-      <h1 className={title}>
+      <h1 css={title}>
         Hello World, this is my first component styled with
         {` `}
         <a href="https://emotion.sh/">emotion</a>!
       </h1>
-      <p className={subtitle}>
+      <p
+        // Styling "inline" via css prop and object styles
+        css={css({
```

Comment:
You don't need a css call here, you can pass the style object directly to the css prop.

Reply:
Removed in 1fc2cee7, thanks.